### DISCUSSION TITLE
Welcome to cloud-carbon-footprint Discussions!

### DISCUSSION THREAD

**Discussion by ccasher** [cloud-carbon-footprint/cloud-carbon-footprint#1039] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

If you do not find what you are looking for here, please check out our Google Groups [discussion forum ](https://groups.google.com/g/cloud-carbon-footprint) as well!

**Comment by jananivarthini455** [comment]
hey hi I am Janani here and interested to contribute to this project

**Reply by ccasher** [reply]
Hi @jananivarthini455, so awesome to hear! Glad you found your way to this project. Please don't hesitate to let us know if there is anything specifically you want to work on, and we will gladly review any of your PR's!

**Comment by jananivarthini455** [comment]
hi @ccasher happy to work with cloud-carbon-footprint project.  please  do
review my PR  (
https://github.com/cloud-carbon-footprint/cloud-carbon-footprint/pull/1328)
and i would love to work upon this issue
(
https://github.com/cloud-carbon-footprint/cloud-carbon-footprint/issues/1217)
can u please elaborate this explain it to me  so that i can able to
understand this and work accordingly.And i want some new screens designs to
wok on.
thank you,
jananivarthini.R
